### ISSUE TITLE
permanent labels

### ISSUE THREAD

**Issue by khera-shanu** [oksktank/react-native-pure-chart#32]
is their a prop (boolean) to make the label in PIe Chart visible all the time

**Comment by zxcv551133** [comment]
Thank you for using react native pure chart.
It will be provided on the next version.

**Comment by sathishu37** [comment]
Yes please provide visible label for the pie chart and moreover this library very simple and effective to use.

**Comment by AppleCkp** [comment]
Hi @sathishu37 ,
Any update for this?
How do i show the permanent percentage in pie chart?

Thanks,
Apple

**Comment by heryvandoro** [comment]
Any update?
